reporting_app/__init__.py

```diff
@@ -453,3 +453,51 @@ def project_status_reports(prj_status):
             table_foot='sum_row_per_column'
         )
     )
+
+
+@app.route('/species')
+@flask_login.login_required
+def all_species():
+    return render_template(
+        'untabbed_datatables.html',
+        'Species',
+        tables=[
+            util.datatable_cfg(
+                'Available species',
+                'species',
+                util.construct_url('species', max_results=1000)
+            ),
+            util.datatable_cfg(
+                'Installed genomes',
+                'genomes',
+                util.construct_url('genomes', max_results=10000)
+            )
+        ]
+    )
+
+
+@app.route('/species/<species>')
+@flask_login.login_required
+def species_page(species):
+    return render_template(
+        'untabbed_datatables.html',
+        species,
+        tables=[
+            util.datatable_cfg(
+                'Summary',
+                'species',
+                util.construct_url('species', where={'name': species}),
+                minimal=True
+            ),
```

Comment:
We could add another table to the spcies page showing the supported genomes for that species.
This would help find out if the user's genome is already supported.
```suggestion
            ),
            util.datatable_cfg(
                'Installed genomes',
                'genomes',
                util.construct_url('genomes', where={'species': species}, max_results=1000),
                minimal=True
            ),
```